src/pages/en/guides/styling.md

```diff
@@ -161,7 +165,32 @@ Use `<style lang="less">` in `.astro` files.
 - **Vue**: `<style lang="scss">`
 - **Svelte**: `<style lang="scss">`
 
-Additionally, PostCSS is supported as an [integration](/en/guides/integrations-guide/).
+## PostCSS
+
+Astro comes with PostCSS included as part of [Vite](https://vitejs.dev/guide/features.html#postcss). To configure PostCSS for your project. Make a `postcss.config.js` in the root of the project directory. Import plugins from [npm](https://www.npmjs.com/search?q=postcss). Using `require()` we apply additional plugins to the configuration file.    
+
+```js
+//./postcss.config.js
+// Import a PostCSS plugin (https://www.postcss.parts/)
+const postcssPresetEnv = require('postcss-preset-env');
+
+module.export={
+  // Plugins as an Array
+  plugins:[
+   postcssPresetEnv({
+     //Plugin options
+   }),
+  autoprefixer({
+    ...options
+  }),
+  ],
+  // Plugins as an object
+  plugins:{
+    postcssPresetEnv:{...options},
+    autoprefixer:{...options},
+  }
+}
+```
 
 
 ---
```

Comment:
I find the horizontal rule after the code block looks a bit off, so maybe remove this too? But this is just me nitpicking.

```suggestion
```

<img width="826" alt="image" src="https://user-images.githubusercontent.com/357379/163021916-62ea4af4-9714-45f9-8cb4-2bb235f7afaa.png">

Reply:
We typically don't use these in the rest of the docs, so I'd probably remove for consistency. The code sample gives a nice separation itself, I think.